src/main/resources/site/content-types/upcomingRelease/upcomingRelease.xml

```diff
@@ -2,11 +2,16 @@
     <display-name>Innhold til publiseringskalender</display-name>
     <super-type>base:structured</super-type>
     <form>
-        <input type="Date" name="date">
```

Comment:
How do you handle existing content with the old field name still present, but no nextRelease field?